.github/workflows/update-cli-reference.yml

```
@@ -0,0 +1,55 @@
+name: Bump Spin
+
+on:
+  workflow_dispatch:
+  # repository_dispatch:
+  #   types:
+  #     - spin-release
+
+jobs:
+  create-pr:
+    name: Create PR with CLI reference bumped
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Download Spin
+        uses: fermyon/actions/spin/setup@v1
```

```suggestion
        uses: fermyon/actions/spin/setup@v1
        with:
          version: canary
```

Just missing a `with:`  line prior to the customizations.  (Don't commit this change, just used it to sketch out.)

---

Oops!  Thanks!